### PR TITLE
Roadmap

### PR DESCRIPTION
**Pull request by wallento**
This branch intends to make our efforts more visible, for example for fundraising.

Please amend to it as needed

### CODE REVIEW

File: site/sitecontent/about.md [hunk 1]
```diff
@@ -20,11 +20,37 @@ As heavy users of OpenCores, we (the FOSSi Foundation) tried everything in our p
...
 We do this to enable everybody to get involved and shape the future of LibreCores with the goal of making it a place where the community can truly feel at home.
 
-Looking ahead in addition to many smaller improvements we want to focus on four major areas.
+#### Basic Features
+
+Some of the most important features we are working on and that will be added in the near future are:
+
+<div style="height: 64px">
+<img src="/img/freepik/hierarchical-structure.png" hspace="24" height="48px" style="float:left"/>
+<p style="overflow: hidden"><b>Organizations.</b> Similar to other platforms such as Github LibreCores can be organized in organizations. The structure is similar to the cores of a user, but with an organization being a collection of users.</p>
```
Comment (olofk) [inline site/sitecontent/about.md]: s/cores/projects as some of the projects can be EDA tools and other things

Comment (olofk) [inline site/sitecontent/about.md]: Comma after Github improves readability

File: site/sitecontent/about.md [hunk 2]
```diff
@@ -20,11 +20,37 @@ As heavy users of OpenCores, we (the FOSSi Foundation) tried everything in our p
...
+</div>
+
+<div style="height: 64px">
+<img src="/img/freepik/label-right-arrow-outline.png" align="left" hspace="24" height="48px" />
+<p style="overflow: hidden"><b>Tags.</b> A core can be tagged with keywords, such as a design language, an interface, an ISA etc. This makes it easier for other users to find what they are searching for.</p>
```
Comment (olofk) [inline site/sitecontent/about.md]: s/core/project

File: site/sitecontent/about.md [hunk 3]
```diff
@@ -20,11 +20,37 @@ As heavy users of OpenCores, we (the FOSSi Foundation) tried everything in our p
...
+</div>
+
+<div style="height: 64px">
+<img src="/img/freepik/rating.png" align="left" hspace="24" height="48px" />
+<p style="overflow: hidden"><b>Reviews and Ratings.</b> We envision a mix of scale-based ratings and an optional comment. The goal is that other users get a better understanding of how other users percepted this core before.</p>
```
Comment (olofk) [inline site/sitecontent/about.md]: s/core/project

File: site/sitecontent/about.md [hunk 4]
```diff
@@ -20,11 +20,37 @@ As heavy users of OpenCores, we (the FOSSi Foundation) tried everything in our p
...
+</div>
+
+<div style="height: 64px">
+<p style="float:left"><img src="/img/freepik/molecular-bond.png" hspace="24" height="48px" /></p>
+<p style="overflow: hidden"><b>Core Relations.</b> Cores rarely stand for themselves, but are part of a larger collection, are forked from another core or depend on other cores.</p>
```
Comment (olofk) [inline site/sitecontent/about.md]: s/core/project

**Comment by wallento** [comment]
ready to merge for me

**Pull request closed without merging**